lib/msal-node/docs/initialize-public-client-application.md

```diff
@@ -40,22 +35,12 @@ const pca = new msal.PublicClientApplication(publicClientConfig);
 
 By default, MSAL is configured with the `common` tenant, which is used for multi-tenant applications and applications allowing personal accounts (not B2C).
 ```javascript
-const msalConfig = {
-    auth: {
-        clientId: 'your_client_id',
-        authority: 'https://login.microsoftonline.com/common/'
-    }
-};
+    authority: 'https://login.microsoftonline.com/common/'
```

Comment:
Not sure why these changes were made...?

Reply:
For consistency with the confidential client doc. Otherwise would have to include all of the other config values, although this sections only speaks about authority

Reply:
Got it, read the other doc. Makes sense now.